### ISSUE TITLE
Create Week 1 Issue 1

### ISSUE THREAD

**Issue by lazyplatypus** [bitprj/Intro-To-Serverless#122]
This should **be in its own issue** and be the first thing in week 1 since students should understand what serverless is before starting to code.

_Originally posted by @FifiTeklemedhin in https://github.com/bitprj/Intro-To-Serverless/pull/107#discussion_r626843716_
